openapi.yaml

```diff
@@ -124,7 +121,6 @@ paths:
           description: Activity code.
           type: string
           required: true
-        - $ref: '#/parameters/authorization'
```

Comment:
I think this part is missing: https://github.com/osu-mist/express-api-skeleton/blob/a72e05b8b43af1ee253f142f74d010bae88b73df/openapi.yaml#L20-L26

Can you add it?